src/Backupper/AbstractBackupper.php

```diff
@@ -78,7 +78,18 @@ public function getDestination(): ?string
 
     public function setExcludedTables(array $excludedTables): self
     {
-        $this->excludedTables = $excludedTables;
+        foreach ($excludedTables as $table) {
```

Comment:
I think you should reset `$this->excludedTables` here.

Or the method name should be `::addExcludedTables`. But, if we consider how you use this method in `BackupCommand`, I think it should definitively be a `setExcludedTable` with a reset.

```suggestion
        $this->excludedTables = [];
        foreach ($excludedTables as $table) {
```

Reply:
> If given, overrides excluded tables parameter from bundle configuration.

From the command description for the `excluded-table` option

Reply:
> I think you should reset $this->excludedTables here.

Indeed! I applied your suggestion :pray: